introduction.md

```diff
@@ -0,0 +1,2 @@
+Combining alchemy with ninja skills, I, Maximiliaan Strother III a.k.a. Mdlkxzmcp shall conquer the entire western kingdom!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.